operator/pkg/common/labels/labels.go

```diff
@@ -0,0 +1,89 @@
+package labels
+
+import (
+	"bytes"
+
+	"github.com/pkg/errors"
+	"helm.sh/helm/v3/pkg/kube"
+	"helm.sh/helm/v3/pkg/postrender"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/cli-runtime/pkg/resource"
+	"sigs.k8s.io/yaml"
+)
+
+var defaultLabels = map[string]string{
+	"app.stackrox.io/managed-by": "operator",
+}
+
+// DefaultLabels defines the default labels the operator should set on resources it creates
+func DefaultLabels() map[string]string {
+	labels := make(map[string]string, len(defaultLabels))
+	for k, v := range defaultLabels {
+		labels[k] = v
+	}
+
+	return labels
+}
+
+// NewLabelPostRenderer is a postrenderer for helm operator plugin kube clients to add
+// given labels to each renderered object
+func NewLabelPostRenderer(client kube.Interface, labels map[string]string) postrender.PostRenderer {
```

Comment:
Why do we need a postRenderer? I though helm-operator already added the `app.kubernetes.io/managed-by: Helm` to all objects it creates, and re-triggers reconcile when those change?

Reply:
See below comment.